engine/src/main/java/org/terasology/config/flexible/FlexibleConfigImpl.java

```diff
@@ -48,26 +49,13 @@ public FlexibleConfigImpl(String description) {
 
     /**
      * {@inheritDoc}
+     * @param id
```

Comment:
That may have snuck in, yes. I'll get rid of it. We technically don't need any Javadoc for the FlexibleConfigImpl class since all of its references will be accessed through the FlexibleConfig interface, but I'll leave the `inheritDoc` be for legacy reasons.